addons/mail/populate/discuss/mail_message.py

```diff
@@ -28,4 +40,11 @@ def _populate(self, size):
                         "res_id": channel.id,
                     }
                 )
-        return res + self.env["mail.message"].create(messages)
+        res = self.env["mail.message"]
+        batches = [messages[i : i + 1000] for i in range(0, len(messages), 1000)]
+        count = 0
+        for batch in batches:
+            count += len(batch)
+            _logger.info("Batch of mail.message for discuss.channel: %s/%s", count, len(messages))
+            res += self.env["mail.message"].create(batch)
```

Comment:
couldn't it be like that since we are already in `mail.message`?
```suggestion
            res += self.create(batch)
```